src/config/i18nConfig.js

```
@@ -0,0 +1,31 @@
+require('dotenv').config();
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import/no-self-import)